.travis.yml

```diff
@@ -33,6 +33,8 @@ jobs:
   exclude:
     - env: MDTRAJ="dev"
       python: "3.6"
+    - env: MDTRAJ="dev"
+      python: "3.7"
```

Comment:
Wouldn't a single `include` (MDTRAJ="dev", python="3.8") be better than 2 `exclude`s? Just a mater of opinion, feel free to ignore.

Reply:
I agreed, and tried it, and the build matrix was WTF weird. See https://travis-ci.com/github/dwhswenson/contact_map/builds/174036372.

Build 2313.4, which says it is deploy testpypi, is actually the extra test with `MDTRAJ="dev"`. So it does what we want, but looks too weird. I'm too busy with other stuff to raise an issue with the Travis people right now, but it might be a bug in how build stages plays with explicit includes. At the least, I find the behavior unexpected. Could be related to different "merging" styles that Travis uses, too.

Reply:
huh, that is very weird indeed, fine by me to leave it for now